src/components/AdminSettings.vue

```diff
@@ -161,6 +161,143 @@
 				</div>
 			</div>
 		</div>
+		<div class="group-folder-setup">
+			<FormHeading index="4"
+				:is-group-folder-setup-heading="true"
+				:title="t('integration_openproject', 'Managed project folders (recommended)')"
+				:is-setup-complete-without-group-folders="isSetupCompleteWithoutGroupFolders"
+				:is-group-folder-setup-correct-after-setup="isGroupFolderSetupCorrectAfterSetup"
+				:is-complete="isGroupFolderSetupCompleted"
+				:is-disabled="isGroupFolderSetUpInDisableMode" />
+			<div v-if="showDefaultManagedFolders">
+				<div v-if="isGroupFolderSetupFormInEdit">
+					<NcCheckboxRadioSwitch type="switch" :checked.sync="isProjectFolderSwitchEnabled" @update:checked="changeGroupFolderSetUpState">
+						<b>{{ t('integration_openproject', 'Automatically managed folders') }}</b>
+					</NcCheckboxRadioSwitch>
+					<div v-if="isProjectFolderSwitchEnabled === false" class="complete-without-groupfolders">
+						<p class="group-folder-description">
+							{{
+								t('integration_openproject', 'We recommend using this functionality but is not mandatory. Please activate in case you want to use the automatic creation and management of project folders.')
```

Comment:
I think `it` is also necessary here, I've commented in figma as well
```suggestion
								t('integration_openproject', 'We recommend using this functionality but it is not mandatory. Please activate in case you want to use the automatic creation and management of project folders.')
```